frontend/src/components/VehicleDetailsPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom';
+import '../styles/vehiclesPage.css'
+
+export function VehicleDetailsPage() {
+    const location = useLocation();
+
+    const vehicle = location.state?.vehicle;
+
+    const vehicles = location.state?.vehicles;
+    const startDate = location.state?.startDate;
+    const startTime = location.state?.startTime;
+    const endDate = location.state?.endDate;
+    const endTime = location.state?.endTime;
+    history.pushState({ startDate, startTime, endDate, endTime, vehicles }, '', '/vehicles')
+
+    return (
+        <div className='vehicle'>
```

Comment:
Ten element można wyciągnąć do osobnego komponentu, bo w pliku VehiclesListPage.jsx jest praktycznie to samo. Wyświetlanie dodatkowych informacji o samochodzie (np. nr rejestracji) można ograć dodatkowym parametrem -> extended czy coś)

Reply:
jest